apps/OboeTester/app/src/main/java/com/mobileer/oboetester/IntentBasedTestSupport.java

```diff
@@ -67,23 +67,27 @@ public class IntentBasedTestSupport {
     public static final String VALUE_VOLUME_TYPE_SYSTEM = "system";
     public static final String VALUE_VOLUME_TYPE_VOICE_CALL = "voice_call";
 
-    public static final String KEY_CHANNEL_MASK = "channel_mask";
+    public static final String KEY_IN_CHANNEL_MASK = "in_channel_mask";
+    public static final String KEY_OUT_CHANNEL_MASK = "out_channel_mask";
     public static final String VALUE_CHANNEL_MONO = "mono";
     public static final String VALUE_CHANNEL_STEREO = "stereo";
     public static final String VALUE_CHANNEL_2POINT1 = "2.1";
     public static final String VALUE_CHANNEL_TRI = "tri";
     public static final String VALUE_CHANNEL_TRI_BACK = "triBack";
+    public static final String VALUE_CHANNEL_TRI_BACK_LOWERCASE = "triback";
     public static final String VALUE_CHANNEL_3POINT1 = "3.1";
     public static final String VALUE_CHANNEL_2POINT0POINT2 = "2.0.2";
     public static final String VALUE_CHANNEL_2POINT1POINT2 = "2.1.2";
     public static final String VALUE_CHANNEL_3POINT0POINT2 = "3.0.2";
     public static final String VALUE_CHANNEL_3POINT1POINT2 = "3.1.2";
     public static final String VALUE_CHANNEL_QUAD = "quad";
-    public static final String VALUE_CHANNEL_QUAD_SIDE = "quadside";
+    public static final String VALUE_CHANNEL_QUAD_SIDE = "quadSide";
+    public static final String VALUE_CHANNEL_QUAD_SIDE_LOWERCASE = "quadside";
     public static final String VALUE_CHANNEL_SURROUND = "surround";
```

Comment:
nit, the lowercase string can be private. BTW, does the following code work?

```
public static final String VALUE_CHANNEL_QUAD_SIDE_LOWERCASE = VALUE_CHANNEL_QUAD_SIDE.toLowerCase();
```

Reply:
Doesn't work.

~~~
/Users/robertwu/Documents/oboe2/oboe/apps/OboeTester/app/src/main/java/com/mobileer/oboetester/IntentBasedTestSupport.java:202: error: constant string expression required
            case VALUE_CHANNEL_TRI_BACK_LOWERCASE:
                 ^
~~~